### DISCUSSION TITLE
[Vanilla Bug] Make side-specific mix-files take priority

### DISCUSSION THREAD

**Discussion by Bittah** [Vinifera-Developers/Vinifera#1386] [Feature Requests]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I have attached as much information as possible *(screenshots, videos, links, etc)*.

### Description

Currently the files in the side-specific mix-files don't take priority, which means that a file that for example exists in both sidec##.mix and ecache##.mix will only be read from ecache##.mix and are only downsides to this.
If the side-specific mix-files would take priority it'd be possible to for example have one file in ecache##.mix that's shared by 3 factions and to then place another file with the same name in for sidec04.mix so that only the 4th faction will use the different file.

**Comment by CCHyper** [comment]
I believe this is because the game does the second pass before it loads these MIX files, but changing this to perform a third pass after they are loaded might introduce a 2-5 seconds loading time increase.
